### ISSUE TITLE
Disable one rule for only one line: trailing pragma makes line too long

### ISSUE THREAD

**Issue by lafrech** [PyCQA/pycodestyle#683]
Hi. I'm trying to ignore a pylint error for a single line and I don't know how it should be done.

Example use case:

```python
        if self._wrapper_class is not None:
            data = self._wrapper_class(data)
            [...]
```

The `data = ...` line triggers a `not-callable` error I would like to mask.

I'm not sure how to do that.

This triggers a `line-too-long` error, unless I also disable `line-too-long` in the line, which makes it even longer (in my real-life use case, data is a longer word and the line is more than 80 characters):

```python
        if self._wrapper_class is not None:
            data = self._wrapper_class(data)  # pylint: disable=not-callable
            [...]
```

This disables the warning for the whole `if` context:

```python
        if self._wrapper_class is not None:
            # pylint: disable=not-callable
            data = self._wrapper_class(data)
            [...]
```

This if functionally correct but a bit too verbose:

```python
        if self._wrapper_class is not None:
            # pylint: disable=not-callable
            data = self._wrapper_class(data)
            # pylint: enable=not-callable
            [...]
```

This was discussed a while ago on [a mailing-list thread](http://python-projects.logilab.narkive.com/P3Cl687J/disabling-warning-for-one-long-line-only-potential-ppylint-bug).

I guess this is bound to happen as long as the only way to disable a warning for a single line is to add a trailing pragma, especially since pylint supports disable by (long) name rather than (short) code.

Shouldn't there be some sort of ignore-for-next-statement pragma?

```python
        if self._wrapper_class is not None:
            # pylint: disable-next=not-callable
            data = self._wrapper_class(data)
            [...]
```

**Comment by sigmavirus24** [comment]
Hi there!

This isn't the pylint project. To file a bug for pylint, please go to [pylint](/pycqa/pylint/issues).

Cheers!
Ian

**Comment by lafrech** [comment]
Oops, my bad. Sorry about the confusion.

**Comment by sigmavirus24** [comment]
No worries! :)
